scripts/usb_reset.py

```diff
@@ -200,7 +200,23 @@ def deny_device(path, domid):
     _device_ctl(path, domid, False)
 
 
-def setup_cgroup(domid, pid):
+def setup_cgroup(domid, pid):  # type:(str, str) -> None
```

Comment:
The type annotation of the `domid` and `pid` should be `int` as follows:
```
    attach.add_argument("-d", dest="domid", type=int, required=True,
                        help="specify the domid of the VM")
    attach.add_argument("-p", dest="pid", type=int, required=True,
                        help="the process id of QEMU")
```

Reply:
Thanks for the heads-up, but the code is agnostic to both types, so it's fine.
I added a suggestion ready to commit it nonetheless, but it is not really a problem either way.

My suggested change is below, with my approval either way, with or without it.